vcnc-server/vcnc-sampler/app.js

```diff
@@ -87,31 +84,51 @@ function requestHandler(request, response) {
   response.end();
 }
 
-const server = http.createServer(requestHandler);
-server.setTimeout(5000, () => {
-});
+// Create server
+//
+let server;
```

Comment:
'server' is only used _inside_ of function startServer.

You should delete line 89 and make line 91 

  const server = http.createServer(requestHandler);

The principles here are (1) use 'let' only if you can't use 'const'; (2) use a smaller scope rather than a larger scope.